.github/workflows/test.yaml

```diff
@@ -20,6 +20,12 @@ jobs:
           go get github.com/mattn/goveralls
           go get github.com/go-playground/validator/v10
 
+
+      - name: Workaround
```

Comment:
Build failed that it couldn't find the lib in the goroot/lib even through its in the go.mod file... Couldn't reproduce locally either

Reply:
Could you add such comment to the yaml file? We probably dont remember this in 2 weeks already;)